addons/web/static/src/views/list/list_renderer.js

```diff
@@ -1387,6 +1392,50 @@ export class ListRenderer extends Component {
                     toFocus = buttons[index - 1] || currentButton;
                 }
                 break;
+            case "shift+arrowdown": {
+                const recordIndex = this.props.list.records.indexOf(record);
+                if (recordIndex >= this.props.list.records.length - 1) {
+                    return false;
+                }
+                const nextRecord = this.props.list.records[recordIndex + 1];
+
+                const isExpanding = this.firstArrowShiftRecord.index < recordIndex + 1;
+                if (isExpanding) {
+                    nextRecord.toggleSelection(this.firstArrowShiftRecord.record.selected);
+                } else {
+                    record.toggleSelection(false);
+                }
+                this.lastCheckedRecord = nextRecord;
+                toFocus = this.findFocusFutureCell(cell, cellIsInGroupRow, "down");
+                break;
+            }
+            case "shift+arrowup": {
+                const recordIndex = this.props.list.records.indexOf(record);
+                if (recordIndex <= 0) {
+                    return false;
+                }
+                const nextRecord = this.props.list.records[recordIndex - 1];
+
+                const isExpanding = this.firstArrowShiftRecord.index > recordIndex - 1;
+                if (isExpanding) {
+                    nextRecord.toggleSelection(this.firstArrowShiftRecord.record.selected);
+                } else {
+                    record.toggleSelection(false);
+                }
+                this.lastCheckedRecord = nextRecord;
+                toFocus = this.findFocusFutureCell(cell, cellIsInGroupRow, "up");
+                break;
+            }
+            case "shift+ ":
```

Comment:
this should not work:
- if it is there's an issue to fix in the hotkey service :-)
- if it isn't you did not test this use case ^^

the correct hotkey string should be "shift+space"